src/workerd/io/worker.c++

```diff
@@ -1546,6 +1546,9 @@ Worker::Worker(kj::Own<const Script> scriptParam,
                             } else if (handle == entrypointClasses.workerEntrypoint) {
                               impl->statelessClasses.insert(kj::mv(handler.name), kj::mv(cls));
                               return;
+                            } else if (handle == entrypointClasses.workflow) {
+                              impl->statelessClasses.insert(kj::mv(handler.name), kj::mv(cls));
```

Comment:
We probably want to extend `validateHandlers()` such that it reports `Workflow` classes differently from regular `WorkerEntrypoint`s. This way, the config service will be able to distinguish between workflow entrypoints and RPC entrypoints, so that it can detect when someone has defined a Workflow and prompt them to configure it as such, and also block people from creating service bindings to workflows (which don't make sense).